dkg/pedersen/types/messages.go

```diff
@@ -65,6 +65,81 @@ func (s Start) Serialize(ctx serde.Context) ([]byte, error) {
 	return data, nil
 }
 
+// ResharingRequest is the message the initiator of the resharing protocol should send to all the
```

Comment:
Comments not formatted at 80 chars. Please ensure that your comment are well formatted:

```
// ResharingRequest is the message the initiator of the resharing protocol
// should send to all the old nodes.
```